xcube/webapi/auth.py

```diff
@@ -169,28 +193,39 @@ def get_id_token(self, require_auth: bool = False) \
                 )
```

Comment:
The message in the previous lines is somewhat misleading: you didn't receive an access token yet. Maybe write something like: "Authentication required, which is not supported by this server"

Reply:
Which lines?

Reply:
Sorry, the lines I was referring to are not part of this PR. Lines 190-192.